.gitattributes

```diff
@@ -0,0 +1,9 @@
+# set default to auto
```

Comment:
Why do we need it ?

Reply:
This config is used in `kubeflow/notebooks` and other repos to avoid CRLF (windows) line endings from being committed to the repo.

It also ensures that files are checked out with their correct line endings (mostly only a problem when users check out on Windows).

For example, `.sh` and `.yaml` files are important because users can check out the repo on a Windows machine (and the files get CRLF) and then if they try and apply the manifests to a WSL cluster, they will bug out because the Linux will not understand the line endings.

The most common cause of this problem is around Pod volume-mounted `.sh` scripts, or ones that are copied into Docker images, because if they have the wrong line endings it makes everything break in confusing ways.